carmajava/launch/environment.launch

```diff
@@ -33,6 +33,124 @@
   <!-- World Model Updater -->
   <include file="$(find carma_wm_ctrl)/launch/carma_wm_broadcaster.launch"/>
 
-  <!-- TODO Object Tracking and World Model go here -->
+  <!-- Object Tracking and World Model -->
+
+  <!-- vision_darknet_detect -->
+  <group>
+  <include file="$(find vision_darknet_detect)/launch/vision_yolo3_detect.launch">
+  <arg name="gpu_device_id" default="0"/>
+  <arg name="score_threshold" default="0.30"/>
+  <arg name="nms_threshold" default="0.30"/>
+
+  <arg name="network_definition_file" default="$(find vision_darknet_detect)/darknet/cfg/yolov3.cfg"/>
+  <arg name="pretrained_model_file" default="$(find vision_darknet_detect)/darknet/data/yolov3.weights"/>
```

Comment:
Probably better to point both of these paths to /opt/carma/yolo/ Then the volume in the docker compose could just be /opt/carma/yolo:/opt/carma/yolo